community/cypher/cypher/src/test/scala/org/neo4j/cypher/ExecutionEngineTest.scala

```diff
@@ -1062,19 +1061,18 @@ order by a.COL1""")
     val planningListener = PlanningListener()
     kernelMonitors.addMonitorListener(planningListener)
 
-    createLabeledNode("Dog")
-    (0 until 400).foreach { _ => createLabeledNode("Person") }
+    (0 until 100).foreach { _ => createLabeledNode("Person") }
     //WHEN
-    eengine.execute(s"match (n:Person:Dog) return n").toList
+    eengine.execute(s"match (n:Person) return n").toList
     planningListener.planRequests.toSeq should equal(Seq(
-      s"match (n:Person:Dog) return n"
+      s"match (n:Person) return n"
     ))
-    (0 until 500).foreach { _ => createLabeledNode("Dog") }
```

Comment:
This test asserted that a change in the cardinality of nodes with label `Dog` of factor 500 is considered small enough. In this case the idea was probably that doing a label scan on `Dog` still makes sense since it is not much better to do the label scan on `Person`. However, the way the way we succeeded with this was basically to never replan.